src/SourceBuild/content/repo-projects/wpf.proj

```diff
@@ -8,7 +8,7 @@
     <BuildArgs>$(BuildArgs) $(FlagParameterPrefix)warnAsError $(ArcadeFalseBoolBuildArg)</BuildArgs>
     <!-- TODO setting Platform shouldn't be necesary: https://github.com/dotnet/source-build/issues/4314 -->
     <BuildArgs>$(BuildArgs) /p:Platform=$(TargetArchitecture)</BuildArgs>
-    <BuildArgs>$(BuildArgs) /p:BuildWithNetFrameworkHostedCompiler=false</BuildArgs>
+    <BuildArgs>$(BuildArgs) /p:BuildWithNetFrameworkHostedCompiler=true</BuildArgs>
```

Comment:
Hm, I hope this won't break something, I guess there was a reason this was disabled previously.

One reason might be that it previously used PackageReference so it didn't work with CPM, but now it uses PackageDownload, so that's fixed.

But also in wpf .net framework builds (which generate wpftmp projects), the toolset compiler doesn't work (PackageDownload is broken there). Not sure if that's relevant here.

Reply:
That was before we had the build-server shutdown support for the framework hosted compiler. CI green so thinks look good.